rewards/utils/emission_utils.py

```diff
@@ -84,3 +122,16 @@ def parse_schedule(schedule) -> Schedule:
         schedule[4],
         schedule[5],
     )
+
+
+def parse_nft_weight_schedule(weight_schedule: List) -> NFTWeightSchedule:
```

Comment:
Can we just add a comment here that explains what the values we're passing here are?  Have to dig into the ABI or class definition to be clear which isn't ideal.